### PR TITLE
update movies function

### PR DESCRIPTION
**Pull request by fkgdmk**

### CODE REVIEW

File: src/main/java/com/outofbounds/empire/EmpireApplication.java [hunk 1]
```diff
@@ -8,5 +8,7 @@
 
 	public static void main(String[] args) {
 		SpringApplication.run(EmpireApplication.class, args);
+
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/EmpireApplication.java:11]: Remove spacing

File: src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java [hunk 2]
```diff
@@ -18,13 +18,38 @@
      * Basic route for movies
      * @return Movie
      */
+
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java]: Remove spacing

File: src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java [hunk 3]
```diff
@@ -18,13 +18,38 @@
...
     @CrossOrigin(origins = "http://localhost:8000")
     @RequestMapping(method = RequestMethod.GET, value = "/movies")
-    public @ResponseBody
-    List<Movie> movies() {
+    public @ResponseBody List<Movie> movies() {
         return movieRepository.findAll();
     }
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java:26]: Remove spacing

File: src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java [hunk 4]
```diff
@@ -18,13 +18,38 @@
...
 
+
+    @RequestMapping(method = RequestMethod.PUT, value = "/movies/{id}")
+    public @ResponseBody
+    Movie updateMovie(@PathVariable int id,
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java]: New after "("

File: src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java [hunk 5]
```diff
@@ -18,13 +18,38 @@
...
+                      @RequestParam (required = false) String title,
+                      @RequestParam (required = false) String description,
+                      @RequestParam (required = false) String category,
+                      @RequestParam (required = false) Integer ageLimit,
+                      @RequestParam (required = false) Integer price)
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java]: new line before ")"

File: src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java [hunk 6]
```diff
@@ -18,13 +18,38 @@
...
+        Movie movie = movieRepository.findById(id);
+
+        try {
+            movie.setTitle(title);
+            movie.setDescription(description);
+            movie.setCategory(category);
+            movie.setAgeLimit(ageLimit);
+            movie.setPrice(price);
+        } catch (NullPointerException e) {
+
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java]: Remove spacing here

File: src/main/java/com/outofbounds/empire/Movies/Models/Movie.java [hunk 7]
```diff
@@ -27,7 +27,7 @@
     private String imageUrl;
 
     @Column(name = "price")
-    private float price;
+    private int price;
 
     public Movie(String title, String description, int ageLimit, String category, float price, String imageUrl) {
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Movies/Models/Movie.java:32]: The price is still float

Comment (jasonkellydk) [reply src/main/java/com/outofbounds/empire/Movies/Models/Movie.java:32]: Change to int

File: src/main/java/com/outofbounds/empire/Movies/Repositories/MovieRepository.java [hunk 8]
```diff
@@ -1,5 +1,4 @@
 package com.outofbounds.empire.Movies.Repositories;
-
```
Comment (jasonkellydk) [inline src/main/java/com/outofbounds/empire/Movies/Repositories/MovieRepository.java]: Add spacing

File: src/test/java/com/outofbounds/empire/Movies/Controllers/MovieControllerTest.java [hunk 9]
```diff
@@ -0,0 +1,21 @@
+package com.outofbounds.empire.Movies.Controllers;
+
+import com.outofbounds.empire.Movies.Models.Movie;
+import org.junit.Test;
+
```
Comment (jasonkellydk) [inline src/test/java/com/outofbounds/empire/Movies/Controllers/MovieControllerTest.java]: Remove spacing

File: src/test/java/com/outofbounds/empire/Movies/Controllers/MovieControllerTest.java [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+import static org.junit.Assert.*;
+
+/**
+ * Created by Fredrik on 26-09-2017.
+ */
+public class MovieControllerTest
+{
```
Comment (jasonkellydk) [inline src/test/java/com/outofbounds/empire/Movies/Controllers/MovieControllerTest.java]: No newline { move it to same line

File: src/test/java/com/outofbounds/empire/Movies/Controllers/MovieControllerTest.java [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+    @Test
+    public void updateTitle() throws Exception {
+
+//        Movie m = new Movie();
```
Comment (jasonkellydk) [inline src/test/java/com/outofbounds/empire/Movies/Controllers/MovieControllerTest.java]: Make a TODO comment if its not done

File: src/main/java/com/outofbounds/empire/EmpireApplication.java (hunk 1 above)
Comment (fkgdmk) [reply src/main/java/com/outofbounds/empire/EmpireApplication.java:11]: done

File: src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java (hunk 2 above)
Comment (fkgdmk) [reply src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java]: done

File: src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java (hunk 4 above)
Comment (fkgdmk) [reply src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java]: done

File: src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java (hunk 5 above)
Comment (fkgdmk) [reply src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java]: done

File: src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java (hunk 6 above)
Comment (fkgdmk) [reply src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java]: done

File: src/main/java/com/outofbounds/empire/Movies/Repositories/MovieRepository.java (hunk 8 above)
Comment (fkgdmk) [reply src/main/java/com/outofbounds/empire/Movies/Repositories/MovieRepository.java]: done

File: src/test/java/com/outofbounds/empire/Movies/Controllers/MovieControllerTest.java (hunk 10 above)
Comment (fkgdmk) [reply src/test/java/com/outofbounds/empire/Movies/Controllers/MovieControllerTest.java]: done

File: src/test/java/com/outofbounds/empire/Movies/Controllers/MovieControllerTest.java (hunk 11 above)
Comment (fkgdmk) [reply src/test/java/com/outofbounds/empire/Movies/Controllers/MovieControllerTest.java]: done

File: src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java (hunk 3 above)
Comment (fkgdmk) [reply src/main/java/com/outofbounds/empire/Movies/Controllers/MovieController.java:26]: done

File: src/main/java/com/outofbounds/empire/Movies/Models/Movie.java (hunk 7 above)
Comment (fkgdmk) [reply src/main/java/com/outofbounds/empire/Movies/Models/Movie.java:32]: done

**Pull request merged**